purchase_quick/models/purchase_order.py

```diff
@@ -49,10 +51,13 @@ def _get_quick_line(self, product):
         return result
 
     def _get_quick_line_qty_vals(self, product):
-        return {
-            "product_uom": product.quick_uom_id.id,
-            "product_qty": product.qty_to_process,
-        }
+        return OrderedDict(
+            {
+                "product_id": None,
+                "product_uom": product.quick_uom_id.id,
+                "product_qty": product.qty_to_process,
+            }
+        )
```

Comment:
Insertion order is guaranteed for regular Python dicts since [Python 3.7](https://docs.python.org/3/library/stdtypes.html#dict.popitem). `OrderedDict` is not needed here.

Reply:
done.